tests/conftest.py

```diff
@@ -0,0 +1,67 @@
+"""Copyright (c) 2017, Anthony Scopatz
+All rights reserved."""
+import json
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+
+import pytest
+
+from regolith.fsclient import dump_yaml
+from regolith.schemas import EXEMPLARS
+
+
+@pytest.fixture(scope='session')
+def make_db():
+    """A test fixutre that creates and destroys a git repo in a temporary
+    directory.
+    This will yield the path to the repo.
+    """
+    cwd = os.getcwd()
+    name = 'regolith_fake'
+    repo = os.path.join(tempfile.gettempdir(), name)
+    if os.path.exists(repo):
+        rmtree(repo)
+    subprocess.run(['git', 'init', repo])
+    os.chdir(repo)
+    with open('README', 'w') as f:
+        f.write('testing ' + name)
+    with open('regolithrc.json', 'w') as f:
+        json.dump({"groupname": "test",
+                   "databases": [
+                       {"name": "test",
+                        'url': repo,
+                        "public": True,
+                        "path": "db",
+                        "local": True}
+                   ]}, f)
+    os.mkdir('db')
+    # Write collection docs
+    for coll, example in EXEMPLARS.items():
+        dump_yaml('db/{}.yaml'.format(coll), {example['_id']: example})
+    subprocess.run(['git', 'add', '.'])
+    subprocess.run(['git', 'commit', '-am', 'Initial readme'])
+    yield repo
+    os.chdir(cwd)
+    rmtree(repo)
+
+
+def rmtree(dirname):
```

Comment:
maybe this should go into tools.py?

Reply:
Yea, I vended this from rever.

Reply:
OK no worries, let's call this non-blocking then

Reply:
At some point we might a) have rever as a req, b) move this to xonsh.

Reply:
> a) have rever as a req,

really?

Reply:
I mean I prefer to move this to xonsh or maybe have a xonsh-utils package with stuff like this? It seems like it would be useful.

Reply:
Sure.  Feel free to add it to xonsh, if something like this isn't there already